test/spec/service/Errors.svc.spec.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
It would be nice to use the Errors.ERROR_CODE.NOT_FOUND instead of the 404 but it's not big deal. Also of not is the mix of single and double quotes. Again not a big deal but the lint errors will go up (will be more strictly enforced after switching to webpack/eslint)